hardhat.config.ts

```diff
@@ -1,34 +1,42 @@
-import { HardhatUserConfig, task } from "hardhat/config";
-import "@nomiclabs/hardhat-etherscan";
-import "@nomiclabs/hardhat-waffle";
-import "@typechain/hardhat";
-import "solidity-coverage";
-
-// This is a sample Hardhat task. To learn how to create your own go to
-// https://hardhat.org/guides/create-task.html
-task("accounts", "Prints the list of accounts", async (taskArgs, hre) => {
-  const accounts = await hre.ethers.getSigners();
-
-  for (const account of accounts) {
-    console.log(account.address);
-  }
-});
-
-// You need to export an object to set up your config
-// Go to https://hardhat.org/config/ to learn more
+import '@nomiclabs/hardhat-ethers';
+import '@nomiclabs/hardhat-etherscan';
+import '@nomiclabs/hardhat-solhint';
+import '@nomiclabs/hardhat-waffle';
+import '@typechain/hardhat';
+import { HardhatUserConfig } from 'hardhat/config';
+import 'solidity-coverage';
 
 const config: HardhatUserConfig = {
-  solidity: "0.8.9",
-  networks: {
-    ropsten: {
-      url: process.env.ROPSTEN_URL || "",
-      accounts:
-        process.env.PRIVATE_KEY !== undefined ? [process.env.PRIVATE_KEY] : [],
+    networks: {
+        ropsten: {
+            url: process.env.ROPSTEN_URL || '',
+            accounts: process.env.PRIVATE_KEY !== undefined ? [process.env.PRIVATE_KEY] : []
+        }
+    },
+    solidity: {
+        compilers: [
+            {
+                version: '0.8.9',
+                settings: {
+                    optimizer: {
+                        enabled: true,
+                        runs: 200
```

Comment:
So you ended up with 200?

Reply:
My bad. I'll update to `2000`

Reply:
TBH, a much more useful thing on the aspect of gas cost would be to implement `mintBatch`, but that's a product decision.